src/test/java/com/wikia/webdriver/testcases/mercurytests/SearchTests.java

```diff
@@ -7,57 +7,83 @@
 import com.wikia.webdriver.common.core.annotations.InBrowser;
 import com.wikia.webdriver.common.core.drivers.Browser;
 import com.wikia.webdriver.common.core.helpers.Emulator;
+import com.wikia.webdriver.common.skin.Skin;
+import com.wikia.webdriver.common.skin.SkinHelper;
 import com.wikia.webdriver.common.templates.NewTestTemplate;
 import com.wikia.webdriver.elements.mercury.components.Search;
 import com.wikia.webdriver.elements.mercury.pages.ArticlePage;
 import com.wikia.webdriver.elements.mercury.pages.SearchResultsPage;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.article.ArticlePageObject;
 
 import org.testng.annotations.Test;
 
 @Execute(onWikia = MercuryWikis.MERCURY_AUTOMATION_TESTING)
-@InBrowser(browser = Browser.CHROME, emulator = Emulator.GOOGLE_NEXUS_5)
+@InBrowser(browser = Browser.CHROME)
 @Test(groups = "MercurySearch")
 public class SearchTests extends NewTestTemplate {
 
   private static final String SEARCH_PHRASE = "Infobox";
   private static final String SEARCH_PHRASE_NO_RESULTS = "AComplexQueryWithNoResults";
 
-  @Test(groups = "mercury_search_navigateToPageUsingSearchSuggestions")
-  public void mercury_search_navigateToPageUsingSearchSuggestions() {
-    String suggestionLink =
+  @InBrowser(emulator = Emulator.GOOGLE_NEXUS_5)
+  @Test(groups = "mercury_search_navigateUsingSearchSuggestionsOnMobile")
+  public void mercury_search_navigateUsingSearchSuggestionsOnMobile() {
+    String clickedSuggestion =
         new ArticlePage()
             .open(MercurySubpages.MAIN_PAGE)
             .getTopBar()
             .openSearch()
-            .typeInSearch(this.SEARCH_PHRASE)
+            .typeInSearch(SEARCH_PHRASE)
+            .clickSearchSuggestion(0);
+
+    Assertion.assertTrue(new SkinHelper(driver).isSkin(Skin.MERCURY));
+    Assertion.assertEquals(clickedSuggestion.toLowerCase(),
+                           new ArticlePage().getHeader().getPageTitle().toLowerCase());
+  }
+
+  @Execute(onWikia = MercuryWikis.MEDIAWIKI_119)
+  @Test(groups = "mercury_search_navigateUsingSearchSuggestionsOnDesktop")
+  @InBrowser(browser = Browser.FIREFOX, browserSize = "1920x1080")
+  public void mercury_search_navigateUsingSearchSuggestionsOnDesktop() {
+    String clickedSuggestion =
+        new ArticlePage()
+            .open("/d/")
```

Comment:
There are PostDetailsPage and PostListPage defined already. If we don't need exactly /d/ url but for example /d/f/1362702 then we can use one of those pages
@SebastianMarzjan ?